OpenEXR/IlmImfExamples/drawImage.cpp

```diff
@@ -144,10 +144,10 @@ zsp (Array2D<half> &gpx, Array2D<float> &zpx, int w, int h,
 	    if (r >= 1)
 		continue;
 
-	    float a = 1;
+	    //float a = 1;
```

Comment:
I assume that fixes an `unused variable` warning? Why not simply remove the line?

Reply:
done